### PR TITLE
Set UnifiedSearchResultNothingFound visibility less messily

### PR DESCRIPTION
**Pull request by claucambra**
Signed-off-by: Claudio Cambra <claudio.cambra@gmail.com>

### CODE REVIEW

**Review by allexzander — APPROVED** [review]

File: src/gui/tray/unifiedsearchresultslistmodel.h [hunk 1]
```diff
@@ -119,6 +122,7 @@ private slots:
 
     QString _searchTerm;
     QString _errorString;
+    bool _waitingForSearchTermEditEnd;
```
Comment (mgallien) [inline src/gui/tray/unifiedsearchresultslistmodel.h:125]: you are missing an initial value for this bool

Comment (claucambra) [reply src/gui/tray/unifiedsearchresultslistmodel.h:125]: Fixed in constructor

Comment (mgallien) [reply src/gui/tray/unifiedsearchresultslistmodel.h:125]: please put it there
that is really easier to read

File: src/gui/tray/Window.qml [hunk 2]
```diff
@@ -685,28 +685,11 @@ Window {
             text: UserModel.currentUser.unifiedSearchResultsListModel.searchTerm
 
             property bool isSearchRunning: UserModel.currentUser.unifiedSearchResultsListModel.isSearchInProgress
+            property bool waitingForSearchTermEditEnd: UserModel.currentUser.unifiedSearchResultsListModel.waitingForSearchTermEditEnd
```
Comment (mgallien) [inline src/gui/tray/Window.qml:688]: I am curious why you do not directly use `UserModel.currentUser.unifiedSearchResultsListModel.waitingForSearchTermEditEnd` in line `692`

Comment (claucambra) [reply src/gui/tray/Window.qml:688]: It's mainly for readability, `UserModel.currentUser.unifiedSearchResultsListModel.waitingForSearchTermEditEnd` is very long

File: src/gui/tray/unifiedsearchresultslistmodel.h (hunk 1 above)
Comment (claucambra) [reply src/gui/tray/unifiedsearchresultslistmodel.h:125]: I am not sure what you mean, would you prefer it be placed in the class declaration or in the constructor?

**Pull request merged**